### PR TITLE
feat: add group feature flag for editing

### PR DESCRIPTION
**Pull request by lekzzz**

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -125,6 +125,11 @@
       "title": "Custom profile schema",
       "default": null
     },
+    "groups.isEnabledEditProfile": {
```
Comment (a23sokolov) [inline docs/feature-flags.schema.json]: обычно пишут `groups .isEditProfileEnabled`

Comment (dmtryyy) [reply docs/feature-flags.schema.json]: `groups.isProfileEditEnabled`

**Comment by dmtryyy** [comment]
What is group's profile? Group only has `about` description

**Comment by lekzzz** [comment]
> What is group's profile? Group only has `about` description

About, title, avatar, short name, permissions

**Comment by lekzzz** [comment]
> This ability could be inferred from the actual member's permissions

For now, all of this fields can be edited only if member has permissions for this. This flag need for enabling this feature at all

**Pull request merged**